Update requirements and docs migration (#11170)
open-metadata/OpenMetadata: openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/DomoDatabase.md

# Requirements

> <!-- to be updated -->

Reviewer: ```suggestion
```

> For metadata ingestion, kindly make sure add alteast `data` scopes to the clientId provided. Question related to scopes, click here.

Reviewer: ```suggestion
For metadata ingestion, kindly make sure to add at least `data` scopes to the clientId provided. For question related to scopes, click here.
```

Reviewer: The link is missing.

You can find further information on the Domo Database connector in the [docs](https://docs.open-metadata.org/connectors/database/domo-database).

## Connection Details

$$section

> ### Client Id $(id="clientId")

Reviewer: ```suggestion
### Client ID $(id="clientId")
```

> Client ID for DOMO. Further information on Client ID can be found [here](https://docs.open-metadata.org/connectors/database/domo-database/troubleshoot#how-to-find-clientid)

Reviewer: ```suggestion
Client ID for DOMO. Further information on Client ID can be found [here](https://docs.open-metadata.org/connectors/database/domo-database/troubleshoot#how-to-find-clientid).
```

> <!-- clientId to be updated -->

Reviewer: ```suggestion
```

$$section
### Access Token $(id="accessToken")

Access token to connect to DOMO. Further information on Client ID can be found [here](https://docs.open-metadata.org/connectors/database/domo-database/troubleshoot#where-to-find-accesstoken)

> <!-- accessToken to be updated -->

Reviewer: ```suggestion
```

$$

$$section

> ### Api Host $(id="apiHost")

Reviewer: ```suggestion
### API Host $(id="apiHost")
```

> API Host to connect to DOMO instance. Default value here is `api.domo.com`

Reviewer: ```suggestion
API Host to connect to DOMO instance. Default: `api.domo.com`.
```

> <!-- apiHost to be updated -->

Reviewer: ```suggestion
```

$$

$$section
### Sandbox Domain $(id="sandboxDomain")

Connect to Sandbox Domain. For example `https://<your>.domo.com`

> <!-- sandboxDomain to be updated -->

Reviewer: ```suggestion
```
